python/tests/res/simulator/test_batch_sim.py

```diff
@@ -409,6 +411,58 @@ def test_stop_sim(self):
             runpath = 'storage/batch_sim/runpath/%s/realisation-0' % case_name
             self.assertTrue(os.path.exists(runpath))
 
+    @tmpdir()
+    def test_ctx_status_failing_jobs(self):
+        def assertQueueStatus(status, idx, final_state_only=False):
```

Comment:
is this function actually doing any assertions?

Reply:
Good catch. It ended up in between implementations, one where the checking was done outside and one where it was done inside the function. I'll fix it!